filebeat/prospector/log/file.go

```diff
@@ -19,10 +19,10 @@ type LogFile struct {
 	done         chan struct{}
 }
 
-func NewLogFile(
+func NewFile(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewFile should have comment or be unexported